src/FileDownloader.ts

```diff
@@ -123,7 +123,7 @@ export default class FileDownloader implements IFileDownloader {
                 return Uri.file(fileDownloadPath);
             };
 
-            return RetryUtility.exponentialRetryAsync(renameDownloadedFileAsyncFn, retries, retryDelayInMs);
+            return RetryUtility.exponentialRetryAsync(renameDownloadedFileAsyncFn, retries * 2, retryDelayInMs);
```

Comment:
Could you please clarify why we are not providing the right `retries` number directly as a setting? It seems strange to take a `retries` parameter but not to respect it. Even if we're using this `retries` value elsewhere, it seems fine to use the same value in both places, or worst case to create two different `retries` variables for each case if separating them is really needed.

Reply:
Sure, I would make a change in our code itself to increase the `retires`.